arm-mobileengagement/2014-12-01/swagger/mobile-engagement.json

```diff
@@ -1289,6 +1289,548 @@
                     }
                 }
             }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MobileEngagement/appcollections/{appCollection}/apps/{appName}/devices/exportJobs": {
+            "get": {
+                "tags": [
+                    "Export",
+                    "List"
+                ],
+                "operationId": "Devices_ListExportJobs",
+                "description": "Get the list of export jobs.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AppCollectionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AppNameParameter"
```

Comment:
Not really. How do we specify non global ? (full disclaimer: i didn't actually wrote this swagger. Just moving it between repos.)